src/cdk/datepicker/datepicker-intl.ts

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Is it possible to track the file change by `git mv`?

Reply:
yes!